@cTab/addons/core/functions/fnc_onDrawbftAndroid.sqf

```diff
@@ -11,12 +11,17 @@ cTabMapScale = ctrlMapScale _cntrlScreen;
 [_cntrlScreen,true] call cTab_fnc_drawUserMarkers;
 private _drawPlayer = [_cntrlScreen,0] call cTab_fnc_drawBftMarkers;
 
+// current position
 _veh = vehicle cTab_player;
 _playerPos = getPosASL _veh;
+_heading = direction _veh;
+if (!(_veh getVariable [QGVAR(enabled),true])) then {
+    _lastKnownTracking = _veh getVariable [QGVAR(lastKnownTracking), [ [0, 0, 0], 0, "000000"]];
+    _playerPos = _lastKnownTracking select 0;
+};
```

Comment:
This pattern is repeated in several places, you might create a function 

ctab_fnc_getPlayerPosition
```
params ["_veh"];
if (_veh getVariable [QGVAR(enabled),true]) then {
    getPosASL _veh
} else {
    (_veh getVariable [QGVAR(lastKnownTracking), [[0, 0, 0], 0, "000000"]]) select 0
}
```